### PR TITLE
initial commit for testing with OpenStudio 3.8.0

### PR DESCRIPTION
**Pull request by DavidGoldwasser**
Current Failing Tests (none)

Pre release
- [ ] Try and fix open issues
- [ ] Make sure xml files updated to 3.8.0 where needed
- [ ] Update readme and change log

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -13,8 +13,11 @@ gemspec
 allow_local = ENV['FAVOR_LOCAL_GEMS']
 
 # Delete when these branchesa are merged and released
-#gem 'openstudio-extension', github: 'NREL/OpenStudio-extension-gem', branch: 'v0.6.0-rc1'
-#gem 'openstudio-standards', '= 0.2.17.rc1', :github => 'NREL/openstudio-standards', :ref => '3.5.0_changes'
+gem 'openstudio-extension', github: 'NREL/OpenStudio-extension-gem', branch: 'wenyi-fix-build-temp', ref: "34a39a24aee2259de"
+gem 'openstudio-standards', github: 'NREL/openstudio-standards', tag: 'v0.6.0.rc2', ref: "1c42110"
+gem 'openstudio_measure_tester', :git => 'https://github.com/NREL/OpenStudio-measure-tester-gem.git', :branch => 'wenyi-dependencies-update'
+gem 'openstudio-workflow', :git => 'https://github.com/NREL/OpenStudio-workflow-gem.git', :branch => 'develop', ref: "32126e9b9f6"
+gem 'bcl', :git => 'https://github.com/wenyikuang/bcl-gem.git', :branch => 'develop'
```
Comment (jmarrec) [inline Gemfile:20]: Some of these gems, especially openstudio_measure_tester should be put into the group `:test`. Which wouldn't even try to get rubocop, and therefore json, and wouldn't crash

Comment (DavidGoldwasser) [reply Gemfile:20]: I have not done this before but I'm happy to make this change.

File: openstudio-ee.gemspec [hunk 2]
```diff
@@ -21,15 +21,21 @@ Gem::Specification.new do |spec|
...
-  spec.add_dependency 'openstudio-standards', '~> 0.5.0'
+  spec.add_dependency 'bundler', '~> 2.4.10'
+  spec.add_dependency 'openstudio-extension', '~> 0.8.0'
+  spec.add_dependency 'openstudio-standards', '0.6.1'
+  spec.add_dependency 'openstudio-workflow', '~> 2.4.0'
+  spec.add_dependency 'octokit', '4.18.0' # for change logs
+  spec.add_dependency 'multipart-post', '2.4.0'
+  spec.add_dependency 'parallel', '1.19.1'
+  spec.add_dependency 'regexp_parser', '2.9.0'
+  spec.add_dependency 'addressable', '2.8.1'
```
Comment (kflemin) [inline openstudio-ee.gemspec:38]: @wenyikuang @DavidGoldwasser i don't think we need anything in line 33-38 here...what do you think?

**Review by kflemin — APPROVED** [review]

File: openstudio-ee.gemspec (hunk 2 above)
Comment (wenyikuang) [reply openstudio-ee.gemspec]:
```
  spec.add_dependency 'multipart-post', '2.4.0'
  spec.add_dependency 'regexp_parser', '2.9.0'
  spec.add_dependency 'addressable', '2.8.1'
```
At least for these new imported library, I didn't see any place we used them..

**Pull request merged**